webapp/templates/jury/executable_edit_content.html.twig

```diff
@@ -33,6 +33,12 @@
                        href="{{ path('jury_executable_download_single', {execId: executable.execid, rank: ranks[idx]}) }}">
                         <i class="fas fa-download"></i> Download
                     </a>
+                    {% if is_granted('ROLE_ADMIN') %}
+                        <a class="btn btn-secondary btn-sm"
```

Comment:
No confirmation?

Reply:
Good call, what's the default way to accomplish that shiny modal dialog?

Reply:
* Add `data-ajax-modal` the the link (so `<a href="..." data-ajax-modal ...>` to make it use AJAX.
* Make the delete action respond to both GET and POST. Get will render a form with 'are you sure' or something and POST will do the actual delete.

The form can be simple, just a submit button to the same action. Note that it does need some logic to switch between ajax and not, like [here](https://github.com/DOMjudge/domjudge/blob/37ff2f0b1fa8061e6f910de1041841573cb57c1d/webapp/src/Controller/BaseController.php#L407-L411) because you can still enter the delete URL in your browser yourself.

If you are stuck let me know and I can push to your branch.

Reply:
I added the modal dialog and it sort of works now. Cancel cancels, delete deletes, but .... it somehow redirects to URLs like `http://localhost/domjudge/jury/executables/awk/undefined` and I have no clue why.
One additional data point: if I remove `data-ajax-modal` I am redirected to the right page (but of course a modal dialog is nicer).

Reply:
Redirect after deletion? That’s odd. Let me investigate

Reply:
This is now fixed, see my commit.